File: server/Maze/Models/Page.cs
using System.ComponentModel.DataAnnotations;

namespace Maze.Models
{
    public class Page
    {
        public int PageId { get; set; }
        [Required]
        public byte[] Img { get; set; }
        public string Text { get; set; }
        public int Doors { get; set; }

    }
}

File: client/src/components/Page.js
import React from 'react';
import PropTypes from 'prop-types';

function Page(props) {
  return (
    <React.Fragment>
    <img src="{props.pageId}.PNG" alt="door {props.pageId}"/>
    <h6>{props.text}</h6>

    </React.Fragment>
  )
}

Page.propTypes = {
  pageId: PropTypes.string.isRequired,
  text: PropTypes.string.isRequired
}

export default Page;
//unsure how I'll need to do this with images...